app/mailers/user_notifications.rb

```diff
@@ -80,10 +80,21 @@ def mailing_list(user, opts={})
     @since = opts[:since] || 1.day.ago
     @since_formatted = short_date(@since)
 
-    @new_topic_posts      = Post.mailing_list_new_topics(user, @since).group_by(&:topic) || {}
```

Comment:
Do we use these scope elsewhere in the app? If not they should probably be removed.